wp-content/themes/workingnyc/assets/js/archive-employer-programs-ca356cc.js

```diff
@@ -1009,7 +1010,8 @@
 	          'page',
 	          'per_page',
 	          'orderby',
-	          'order'
+	          'order',
+	          'post_type'
```

Comment:
Yes, that is what history.omit does

Reply:
'post_type' is included in history.omit because all parameters in history.omit are removed from the URL state management, so I will keep 'post_type' in the array